http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=532048

The nfs-utils package in Ubuntu has been patched to reduce the default log
verbosity in the case of expired Kerberos credentials, for the reasons
described here:

https://bugs.launchpad.net/ubuntu/+source/nfs-utils/+bug/293705

Please consider applying the attached patch to Debian as well.

Andrew Pollock said he was submitting this patch upstream, but there doesn't
seem to be any evidence that it's been included yet in 1.2.0.

Steve Langasek <steve.langasek@canonical.com>

--- a/utils/gssd/gssd_proc.c	2010-02-18 23:35:00.000000000 +1100
+++ b/utils/gssd/gssd_proc.c	2010-04-06 16:43:55.000000000 +1000
@@ -638,7 +638,7 @@ do_error_downcall(int k5_fd, uid_t uid, 
 	unsigned int timeout = 0;
 	int	zero = 0;
 
-	printerr(1, "doing error downcall\n");
+	printerr(2, "doing error downcall\n");
 
 	if (WRITE_BYTES(&p, end, uid)) goto out_err;
 	if (WRITE_BYTES(&p, end, timeout)) goto out_err;
@@ -906,7 +906,7 @@ process_krb5_upcall(struct clnt_info *cl
 	int			create_resp = -1;
 	int			err, downcall_err = -EACCES;
 
-	printerr(1, "handling krb5 upcall (%s)\n", clp->dirname);
+	printerr(2, "handling krb5 upcall (%s)\n", clp->dirname);
 
 	if (tgtname) {
 		if (clp->servicename) {
@@ -1066,7 +1066,7 @@ process_spkm3_upcall(struct clnt_info *c
 	}
 
 	if (!authgss_get_private_data(auth, &pd)) {
-		printerr(1, "WARNING: Failed to obtain authentication "
+		printerr(2, "WARNING: Failed to obtain authentication "
 			    "data for user with uid %d for server %s\n",
 			 uid, clp->servername);
 		goto out_return_error;
--- a/utils/gssd/gss_util.c	2010-02-18 23:35:00.000000000 +1100
+++ b/utils/gssd/gss_util.c	2010-04-06 16:43:55.000000000 +1000
@@ -126,7 +126,7 @@ display_status_1(char *m, u_int32_t code
 				"gss_display_status called from %s\n", m);
 			break;
 		} else {
-			printerr(0, "ERROR: GSS-API: (%s) error in %s(): %s\n",
+			printerr(2, "ERROR: GSS-API: (%s) error in %s(): %s\n",
 			    typestr, m, (char *)msg.value);
 		}
 
